src/ResourceManager/Kubernetes/Commands.Kubernetes.Test.Netcore/MergeKubeConfigTests.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Microsoft.Azure.Commands.Kubernetes;
+using Xunit;
+using YamlDotNet.RepresentationModel;
+
+namespace Commands.Kubernetes.Test
+{
+    public class MergedKubeConfigTests
+    {
+        private readonly YamlNode _rootNode;
+
+        public MergedKubeConfigTests()
+        {
+            var merged = ImportCredential.MergeKubeConfig(File.ReadAllText("./Fixtures/config.yaml"), File.ReadAllText("./Fixtures/additional_kube_config.yaml"));
+            _rootNode = RootNodeFromString(merged);
+        }
+
+        [Fact]
+        public void MergedKubeConfigOverwriteClustersTest()
```

Comment:
Use traits to mark these as check-in tests (```[Trait(Category.AcceptanceType, Category.CheckIn)]``` these constants are in the common scenario test assembly